src/app/components/navbar/navbar.component.scss

```diff
@@ -171,13 +170,19 @@ $hellenicblue: rgb(0, 140, 255);
 @media screen and (max-width: 1079px) {
     .navbar {
         margin-left: 0;
-        width: 214%;
+        width: 215vw;
     }
     .intro-bar {
         margin: 0;
-        width: 217%;
+        width: 221vw;
```

Comment:
Using width and height should be avoided where possible. This can be achieved instead by using display flex on the parent and seperating them with justify content. Using these values like this is bad because when zoomed it will distort again i.e different screen resolution phones